nsxt/resource_nsxt_policy_role_binding.go

```diff
@@ -0,0 +1,320 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/aaa"
+	nsxModel "github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+// Only support local user at the moment
+var roleBindingUserTypes = [](string){
+	nsxModel.RoleBinding_TYPE_LOCAL_USER,
+	nsxModel.RoleBinding_TYPE_REMOTE_USER,
+	nsxModel.RoleBinding_TYPE_REMOTE_GROUP,
+	nsxModel.RoleBinding_TYPE_PRINCIPAL_IDENTITY,
+}
+
+var roleBindingIdentitySourceTypes = [](string){
+	nsxModel.RoleBinding_IDENTITY_SOURCE_TYPE_VIDM,
+	nsxModel.RoleBinding_IDENTITY_SOURCE_TYPE_LDAP,
+	nsxModel.RoleBinding_IDENTITY_SOURCE_TYPE_OIDC,
+	nsxModel.RoleBinding_IDENTITY_SOURCE_TYPE_CSP,
+}
+
+func resourceNsxtPolicyUserManagementRoleBinding() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyUserManagementRoleBindingCreate,
+		Read:   resourceNsxtPolicyUserManagementRoleBindingRead,
+		Update: resourceNsxtPolicyUserManagementRoleBindingUpdate,
+		Delete: resourceNsxtPolicyUserManagementRoleBindingDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"display_name": getDataSourceDisplayNameSchema(),
```

Comment:
is there a reason we don't allow controlling `nsx_id` here and in the other policy resource?